### PR TITLE
Update atlas_engine readme to include directions for redeeming hosted atlas api key

### PR DESCRIPTION
**Pull request by kwiersema**
## Context

Add an example curl request for how to query the hosted `atlas-validation.shopifyapps.com` solution after redeeming an api key.

## Approach

...

## Testing

Is the explanation clear and concise?
Does the example curl request work?

## Checklist

- [x] I have added a CHANGELOG entry for this change (or determined that it isn't needed)
- [x] Added Sorbet signatures to new methods I've introduced 
- [x] Commits squashed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -653,3 +653,19 @@ will produce the response:
...
+
+```
+curl --request POST \
+  --url https://atlas-validation.shopifyapps.com/graphql \
+  --header 'Authorization: Bearer {your-api-key}' \
+  --header 'Content-Type: application/json' \
+  --data '{"query":"query validation {\n validation(\n address: {\n address1: \"131 Greene St\"\n address2: \"\"\n city: \"New York\"\n countryCode: US\n provinceCode: \"NY\"\n zip: \"10012\"\n }\n locale: \"EN\"\n ) {\n validationScope\n concerns {\n code\n fieldNames\n suggestionIds\n type\n typeLevel\n message\n }\n suggestions {\n id\n address1\n address2\n city\n province\n provinceCode\n zip\n }\n }\n}","operationName":"validation"}'
+```
+
+Any updates to the `atlas_engine` codebase that are merged into our `main` branch will be deployed to our hosted solution as well.
```
Comment (devanandersen) [inline README.md:671]: We should also link to the terms and conditions here - which has a notice about the RPM/RPS limits, etc - trying to find the link

**Review by devanandersen — APPROVED** [review]
LGTM - would suggest at least a note of some kind about RPS/RPM limits

File: README.md (hunk 1 above)
Comment (celsowhite) [reply README.md:671]: Here is that link https://address-validation.shopify.dev/terms-of-service

**Pull request merged**